app/src/db/migrations/20220822235439_025-alter-form-table-to-add-schedule-col.js

```diff
@@ -0,0 +1,15 @@
+exports.up = function(knex) {
+  return Promise.resolve()
+  // create new collumn to store Schedule (Period) setting
+    .then(() => knex.schema.alterTable('form', table => {
+      table.jsonb('schedule').comment('Form level Schedule settings.');
```

Comment:
is there a reason why you decided to use json for the schedule field instead of regular fields?

Reply:
Because Schedule Object will be deep and big so It's better to save it in a JSON. Do you recommend something else?

Reply:
Can we have a different table for the schedule?

Reply:
Like just for the Schedule Settings? If Yes, Then I will have to take a look then can answer.

Reply:
I would suggest that this table.jsonb('schedule').comment('Form level Schedule settings.'); should be in a different table since it is a 1:Many relationship.